src/eml/main.ml

```diff
@@ -7,7 +7,7 @@
 
 module Command_line :
 sig
-  val parse : unit -> (string * string) list
+  val parse : unit -> (string * string * bool) list
```

Comment:
With the way the code is currently written, there is a global syntax flag for all the files, so the result type should be

```ocaml
(string * string) list * bool
```

However, I think it would be even better to move the code from `process_file` in `eml.ml`, which checks the extension, into the command-line parser, and decide here what syntax each file will use. Then, the result type should be either what this PR has now, or, even more helpfully

```ocaml
(string * string * [ `OCaml | `Reason ]) list
```